EdgeMultiplay/Scripts/DataContracts.cs

```diff
@@ -307,21 +309,24 @@ public class CreateRoomRequest
         public int playerAvatar;
         [DataMember]
         public int maxPlayersPerRoom;
+        [DataMember (EmitDefaultValue = false)]
+        public Hashtable playerTags;
 
-        public CreateRoomRequest(string PlayerName, int PlayerAvatar, int MaxPlayersPerRoom)
+        public CreateRoomRequest(string PlayerName, int PlayerAvatar, int MaxPlayersPerRoom , Hashtable playerTags = null)
         {
             type = "CreateRoom";
             playerId = EdgeManager.gameSession.playerId;
             playerName = PlayerName;
             playerAvatar = PlayerAvatar;
             maxPlayersPerRoom = MaxPlayersPerRoom;
+            this.playerTags = playerTags;
         }
     }
 
-        /// <summary>
-        /// JoinOrCreateRoomRequest is sent to the server to join a room or create a new room if it there is no available rooms
-        /// </summary>
-        [DataContract]
+    /// <summary>
+    /// JoinOrCreateRoomRequest is sent to the server to join a room or create a new room if there are no available rooms
```

Comment:
Ah this answers my question from the node server PR. I think this function is redundant (or the other two functions are redundant). I think it might make more sense to just have JoinRoom have this functionality (ie. Create a room if there are no available rooms). Or maybe get rid of the JoinRoom and CreateRoom cases and only have this case. This can be a future change though, if you agree

Reply:
I think I replied to this comment in the server repo.